tests/Schema/Keywords/TypeTest.php

```diff
@@ -93,4 +93,24 @@ public function invalidDataProvider(): array
             ['integer', 1.0],
         ];
     }
+
+    public function testItValidatesDictionaryTypeGreen(): void
+    {
+        $spec = <<<SPEC
+schema:
+  type: array
+  items:
+    type: string
```

Comment:
`string` type cannot have additional properties. only `object` type is allowed to do so. can you please provide an example test with samples from linked docs in order to check that the problem exists?